bundles/best_practices.rst

```diff
@@ -162,6 +162,15 @@ standard Symfony autoloading instead.
 A bundle should also not embed third-party libraries written in JavaScript,
 CSS or any other language.
 
+Doctrine Entities/Documents
+---------------------------
+
+If the bundle includes Doctrine ORM entities and/or ODM documents, it's
+recommended to define their mapping using XML files stored in ``config/``.
```

Comment:
Two minor things here:

1) Both `DoctrineBundle` and `DoctrineMongoDBBundle` use `Resources/config/doctrine` as their default bundle mapping paths in their respective container extensions, which leads to the next item
2) It doesn't look like either bundle (nor did I find anything in the base container extension in the Doctrine Bridge to cover it) actually supports the newer bundle directory structure (for a bundle I recently released, I had to manually add compiler passes adding the `<bundle_root>/config/doctrine` path)